src/main/java/ToDo.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * ToDos are task objects that only have a description.
+ *
+ * @author AkkFiros
+ */
+public class ToDo extends Task {
+    /**
+     * Constructor for a ToDo task
+     * @param description description of a task that the user inputs
+     */
+    public ToDo(String description) {
+        super(description);
+    }
+
+    /**
+     * toString method for a ToDo task
```

Comment:
Perhaps we can start our method header comments in the form `Returns ...`, `Sends ...`, `Adds ...`